dwio/nimble/velox/tests/VeloxReaderTests.cpp

```diff
@@ -4588,12 +4588,12 @@ TEST_F(VeloxReaderTests, EstimatedRowSizeComplex) {
           .keyType = type->kind(),
           .maxNumKVPerRow = numFeatures,
           .variantNumKV = false,
+          .seed = 1387939242,
```

Comment:
Will this break the synchronization of code?

Reply:
Why? This is part of Nimble. We just need to import as a diff as we usually do, no?

Reply:
Yeah. I'll import it soon. Just making sure CI is green first.